cmd/gctcli/commands.go

```diff
@@ -3589,10 +3483,6 @@ func getTickerStream(c *cli.Context) error {
 		exchangeName = c.Args().First()
 	}
 
-	if !validExchange(exchangeName) {
```

Comment:
The only concern here is that rpc function `GetTickerStream` will return the error `fmt.Errorf("ticker item not found for %s %s %s",
			exchange,
			p,
			a)`
which is less clear than exchange not found. Would you be willing to add
 ```
exch := bot.GetExchangeByName(r.Exchange)
	if exch == nil {
		return nil, ErrExchangeNotFound
	}
``` 
to `GetExchangeTickerStream`?